discovery-service/src/test/resources/api-defs-empty/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
Just out of curiosity, this is there for git to track the directory?

Reply:
Yes, the test needs an empty directory, which was previously provided using the TemporaryFolder rule from Junit 4. There is no equivalent for this in Junit 5 (prior to version 5.4, we are still on 5.3, and even after 5.4, there is TempDir attribute that is still experimental).

Refs:
https://stackoverflow.com/questions/56613895/how-to-get-rid-of-temporaryfolder-rule-in-junit5/56614039
https://stackoverflow.com/questions/115983/how-can-i-add-an-empty-directory-to-a-git-repository